mne/epochs.py

```diff
@@ -1233,16 +1250,24 @@ def plot_image(self, picks=None, sigma=0., vmin=None,
                                  cmap=cmap, fig=fig,
                                  overlay_times=overlay_times)
 
-    @verbose
+    @deprecated('drop_bad method has been renamed drop_bad. '
+                'drop_bad method will be removed in 0.13')
```

Comment:
-> ``'drop_epochs method has been renamed drop_bad. '
 'drop_epochs method will be removed in 0.13'``